src/plugins/injection/README.md

```diff
@@ -0,0 +1,325 @@
+- infos = Information about the injection plugin is below
+- infos/author = Markus Raab <elektra@libelektra.org>
+- infos/licence = BSD
+- infos/needs =
+- infos/provides = inject
+- infos/recommends =
+- infos/placements = presetstorage
+- infos/status = maintained nodep
+- infos/metadata = inject/randSeed inject/structure/sectionRemove inject/structure/sectionDuplicate inject/structure/sectionReallocate inject/semantic inject/resource inject/typo/transposition inject/typo/insertion inject/typo/caseToggle inject/typo/deletion inject/typo/changechar inject/typo/space inject/domain inject/limit/min inject/limit/max
+- infos/description = Injections for configurations
+
+# Injection Plugin
+
+## Introduction
+
+This plugin is used for purposefully alter single or multiple configuration settings in beneath a keyset.
+The plugin searches for metadata 'markers' on every key and depending on the injection, requires additional metadata.
+How the specific injection is carried out can either be random or, if desired, pseudorandom to guarantee reproducability.
+
+## Usage
+
+There up to 6 error types currently with injection types for each.
+1. Structural errors (e.g. missing sections, parameters in wrong sections, omission necessary parameters in section)
```

Comment:
Afaik some markdown renderer need a newline in between so that this enumeration is recognized.